dts/bindings/fs/zephyr,fstab-base.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Common properties for Zephyr filesystem compatibles
+
+include: base.yaml
+
+properties:
+  mount-point:
+    type: string
+    required: true
+    description: |
+      The absolute path used as the file system mount point.
```

Comment:
Seems like this belongs in the property description for the same property in zephyr,fstab.yaml.

Reply:
Actually it's not supposed to be in `fstab`, which is currently just a holder for `zephyr,fstab,foo` entries, each of which has at least these properties.

Reply:
No, actually, it is supposed to be there because it's the child binding (which I couldn't see from the restricted context of the diff) in the review.

Maybe the child binding should be empty?  If there's child-specific properties like this, how do we align the child binding properties with the bindings for the children that also add more properties?

Reply:
Good question. I would say defining a common include like spi-device.yaml and pulling that in from each FS specific compatible binding is probably the way to go.

Reply:
I've removed the child binding and renamed `zephyr,fstab-base.yaml` to `zephyr,fstab-common.yaml`.